Bottom: 7295eaa88c38b5a808f5cdcc82ec0aa947801a02
Top:    478831cc94f48f6b95734cbbbae22ed8365a4642
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 20:53:46 +0000

Refresh of temporarily-disable-rubber

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 23a88ea..c0748c1 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -39,7 +39,6 @@
       are marked
 */
 
-#undef NDEBUG
 #include	<assert.h>
 #include	<stdlib.h>
 #include	<stdio.h>
@@ -937,7 +936,6 @@ insert_vertex_in_seg (struct info *i, struct seg *s, Vector v, double param)
   DEBUGP ("new intersection on segment \"i\" at %#mD\n", v[0], v[1]);
 #endif
   i->node_insert_list = prepend_insert_node_task (i->node_insert_list, s, new_node, param);
-  assert (!s->intersected);
   s->intersected = 1;
   return true;
 }
@@ -1292,8 +1290,6 @@ seg_in_seg_arc_arc (struct info *i, struct seg *s1, struct seg *s2)
       s1->v->radius == s2->v->radius)
     {
       printf ("Cop-out for co-circular arcs for now - they will be better handled by their line approximations. <<<<<<<<<<\n");
-      s1->v->is_round = false;
-      s2->v->is_round = false;
       return seg_in_seg_line_line (i, s1, s2);
     }
 #endif
